### ISSUE TITLE
Background pictures

### ISSUE THREAD

**Issue by Ameseattle** [RabbitPair/colorful_sidepanel_tabs_extension#2]
When U add a background picture it shows up on the option page but won't show in the side panel. If I switch to your sample it works fine. What am I doing wrong?

**Desktop (please complete the following information):**
Windows 11
chrome - Version 127.0.6533.120

**Additional context**
![your pic](https://github.com/user-attachments/assets/25fd07e5-ba43-4194-9b1b-6b0f12ad16de)
![my picture](https://github.com/user-attachments/assets/aed859c3-c7c7-492b-b999-7e2ee95bcf16)

**Comment by ymlluo** [comment]
Can you provide your image URL?

**Comment by Ameseattle** [comment]
it was downloaded from your link on the
      options page:
https://www.pexels.com/photo/photo-of-multicolored-illustration-2832382/
      
      to this folder on my computer:
      C:\Users\amese\Downloads\pexels-anniroenkae-2832382 (1).jpg
      
    the address on my computer after
      installing in the extension is:
file:///C:/Users/amese/Downloads/pexels-anniroenkae-2832382%20(1).jpg
    
      btw- it looks the same no mater where I set image transparency...
      when using your sample, moving image
      transparency slider to the right makes the pic more visible
    
    
    On 8/17/2024 1:34 AM, Ymlluo wrote:
    
    
      
      Can you provide your image URL?
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        You are receiving this because you authored the thread.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/RabbitPair/colorful_sidepanel_tabs_extension/issues/2#issuecomment-2294794459",
"url": "https://github.com/RabbitPair/colorful_sidepanel_tabs_extension/issues/2#issuecomment-2294794459",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by ymlluo** [comment]
I was able to reproduce the issue you mentioned.

When you download an image to your computer, if the file name contains brackets or spaces, it won’t show up when used as a background image. However, this problem doesn’t occur if you use the image’s URL directly.

- **Solution 1:** Rename the local file to `pexels-anniroenkae-2832382.jpg` and use `file:///C:/Users/amese/Downloads/pexels-anniroenkae-2832382.jpg` as the file path.
- **Solution 2:** Use the image URL directly: `https://images.pexels.com/photos/2832382/pexels-photo-2832382.jpeg`. If you want to use another image, just right-click on it and select "Copy Image Address"—like this:  
  ![image](https://github.com/user-attachments/assets/43440311-b859-45e9-bbba-861c040e6a40).

I also noticed a few other things. For example, dark mode might work better with a dark image background. If you’re using light mode and switch the text color to white, some interfaces with a white background could make the text invisible.

Thanks for your feedback! I’m planning to fix this in the next version by allowing image uploads directly from your computer and optimizing the interface display.
